META.d/_summary.yaml

```diff
@@ -0,0 +1,4 @@
+partition: hcp
+schema: 1.1
+summary:
+  owner: team-cloud-core-platform
```

Comment:
I wasn't sure if we wanted to put an owner on this since it was an open-source fork.